src/deployments.json

```diff
@@ -0,0 +1,188 @@
+{
```

Comment:
Nit: I think that since **all** the addresses are the same across the different networks, they should just change on different versions. I would just create some objects like:
```typescript
const v1Addresses = {
  "reality": "0x..01",
  ...
}
```

Then you can probably iterate on the array of known networks (https://github.com/gnosis/zodiac/pull/50/files#diff-d699f4083ec87500be754807462c3cca838b34c782e4dc8e203c332c3950a953R11) and create this big object? That's probably cleaner I think